eng/Xunit3/Microsoft.Testing.Platform.targets

```diff
@@ -78,6 +78,14 @@
       <_ResultsFileToDisplay Condition="!Exists('$(_ResultsFileToDisplay)')">%(TestToRun.ResultsStdOutPath)</_ResultsFileToDisplay>
     </PropertyGroup>
 
+    <!-- Generate a test report -->
+    <Exec Command="pwsh -command &quot;$(RepoRoot)eng/scripts/gha-testreport.ps1 -TestResultsFolder '$(_TestResultDirectory)' -TestSummaryOutputFolder '$(TestResultsLogDir)'&quot;"
+          Condition=" '$(IsGitHubActionsRunner)' == 'true' "
+          WorkingDirectory="$(RepoRoot)"
+          IgnoreExitCode="true"
+          ContinueOnError="WarnAndContinue"
+          />
```

Comment:
This trick allows us generating a report as part of a test run:
![image](https://github.com/user-attachments/assets/a96c19f4-d4b2-4970-bca2-ee445506f2df)

However, I totally realise this may not stand the test of time (considering #8802) if we can't figure out how to bake this into the Arcade SDK.